### ISSUE TITLE
58411447 - The Legend of Korra

### ISSUE THREAD

**Issue by Etokapa** [xenia-project/game-compatibility#1253]
[Marketplace](https://marketplace.xbox.com/en-US/Product/The-Legend-of-Korra/66acd000-77fe-1000-9115-d80258411447)

Tested on https://github.com/xenia-project/xenia/commit/f7ea93148a9ec9ff0ad9edac8c01854bd6153a10

Booted game. Intro logos and title screen run fine.

# Issues:
• Nothing happens when you press Start on the title screen. The game still runs, but you can't progress any further. 

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/3359511/xenia.zip)

# Screenshot(s):
![k](https://user-images.githubusercontent.com/31154606/60670910-6cf69500-9e37-11e9-948b-b9738a160d01.png)

# Labels:
state-title | tech-middleware-criware

**Comment by Margen67** [comment]
Does the game freeze while it's not responding to input?

**Comment by Etokapa** [comment]
No, it's still running at full speed. It's strange because the "Press any button" text fades away, and instead of showing a menu or starting the game, is just sits on the title. It doesn't respond to controls at that point either.

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/86ba17847b79a67afc5bf52cc7fd1aaf5c38d379

Nothing happens when you press Start on the title screen. The game still runs, but you can't progress any further.

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5758178/xenia.zip)

kernel-KeSetCurrentStackPointers

**Comment by chry28san** [comment]
Has the frozen screen problem been resolved?

**Comment by chrysthierry** [comment]
Has the frozen screen problem been resolved?

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/412
